src/BenchmarksClient/Workers/SignalRWorker.cs

```diff
@@ -311,9 +319,16 @@ private void CalculateStatistics()
             Log($"Least Requests per Connection: {min}");
             Log($"Most Requests per Connection: {max}");
 
+            if (_workTimer.ElapsedMilliseconds <= 0)
```

Comment:
What does it do?

Reply:
Prevents the `NaN` and thus the failed SQL query